Helper/Entity/ProductHelper.php

```diff
@@ -1379,9 +687,7 @@ public function canProductBeReindexed($product, $storeId, $isChildProduct = fals
      */
     public function productIsInStock($product, $storeId): bool
```

Comment:
Do we want to deprecate this method as well?

Reply:
I was about to do it then I remembered that this method has its plugin in the inventory module: https://github.com/algolia/algoliasearch-inventory-magento-2/blob/master/Plugin/Helper/ProductHelperPlugin.php#L50

From what I remember, this module was introduced because we wanted to keep compatibility with Magento 2.3 back in the days, I'm wondering if we could not just add the MSI handling directly in the codebase now ?

Reply:
Yes that would be my preference as MSI is pretty much standard as of 2.4. Why make customers install a different module? Maybe we deprecate but don't specifically mention 3.16 in case we don't merge MSI handling until 3.17 for instance. At any rate, this is good validation for us to continue to use this deprecation approach going forward - because if we are using plugins on this stuff then surely other people might as well.